mailserver/mailserver.go

```diff
@@ -296,7 +316,7 @@ func (s *WMailServer) validateRequest(peerID []byte, request *whisper.Envelope)
 
 	if upper < lower {
 		log.Error(fmt.Sprintf("Query range is invalid: from > to (%d > %d)", lower, upper))
-		return false, 0, 0, nil
+		//return false, 0, 0, nil
```

Comment:
Even if we are catching the panic, I think it's worth to stop processing request here, as this is clearly an incorrect request and proper fix would be to communicate this to the caller somehow.

Reply:
Oops, that was part of testing, and not supposed to be committed. Thanks for catching that!